pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/CaseTransformFunctionTest.java

```diff
@@ -112,25 +110,13 @@ private void testCaseQueryWithDoubleResults(String predicate, double[] expectedV
 
   private void testCaseQueryWithStringResults(String predicate, String[] expectedValues) {
     ExpressionContext expression =
-        RequestContextUtils.getExpression(String.format("CASE(%s, 'aaa', 'bbb')", predicate));
+        RequestContextUtils.getExpressionFromSQL(String.format("CASE WHEN %s THEN 'aaa' ELSE 'bbb' END", predicate));
     TransformFunction transformFunction = TransformFunctionFactory.get(expression, _dataSourceMap);
     Assert.assertTrue(transformFunction instanceof CaseTransformFunction);
     Assert.assertEquals(transformFunction.getName(), CaseTransformFunction.FUNCTION_NAME);
     testTransformFunction(transformFunction, expectedValues);
   }
 
-  @Test(dataProvider = "testIllegalArguments", expectedExceptions = {BadQueryRequestException.class})
```

Comment:
why we removed these tests?

Reply:
These illegal arguments are not valid sql expression, and the parser will throw `SqlParseException`. The purpose of this test is not to test the sql parser, thus removed the test.